### ISSUE TITLE
Setup UI

### ISSUE THREAD

**Issue by wardlem** [JBaczuk/bitcoin-script-js#21]
We need a basic frontend setup.  There are a plethora of ways to accomplish this.  I am personally fond of [mithril](https://mithril.js.org/) and I think that it would be a good fit for what we are doing here.  

If we do this right, we can set up a simple static website to host this thing.  That will keep server costs to an absolute minimum.

**Comment by JBaczuk** [comment]
Mithril works for me. I think bitcoin.org is a ruby site, so a static site would be ideal if they wanted to integrate it.

**Comment by ghost** [comment]
@JBaczuk Is this project supposed to include the user interface or should it only provide the script execution logic?

**Comment by JBaczuk** [comment]
You're right, I think the UI would be a separate project, if you want you can create that repo on your account. We could probably host it using github pages, I think.
